controller/monitor/disk_utils.go

```diff
@@ -106,6 +115,10 @@ func getBlockTypeDiskConfig(client *DiskServiceClient, diskName, diskPath string
 		return nil, err
 	}
 
+	if info.State != string(DiskStateReady) {
+		return nil, fmt.Errorf("disk %v is not ready: %v", diskName, info.State)
+	}
```

Comment:
Can we have a more graceful handling? This stat get function will be used by other functions like `CreateDisksFromAnnotation`. Then this failure will lead to all its upper callers' failures, including `syncKubernetesNode`.

Reply:
Addressed. Use `diskconfig.State` instead. Thank you.